shiny/ui/_input_update.py

```diff
@@ -785,10 +785,16 @@ def update_slider(
     min_num = None if min is None else _as_numeric(min)
     max_num = None if max is None else _as_numeric(max)
     step_num = None if step is None else _as_numeric(step)
+    if isinstance(value, tuple):
```

Comment:
It is possible (and likely) for users to pass in a dictionary, especially if they're not using an IDE with type hints.
```suggestion
    if isinstance(value, (tuple, dict)):
```